imx-core-sdk-kotlin-jvm/src/main/kotlin/com/immutable/sdk/StandardStarkSigner.kt

```diff
@@ -9,7 +10,9 @@ import java.util.concurrent.CompletableFuture
 /**
  * A simple implementation of [StarkSigner] that holds the [ECKeyPair] in memory.
  */
-class StandardStarkSigner(@VisibleForTesting internal val keyPair: ECKeyPair) : StarkSigner {
+class StandardStarkSigner(privateKey: String) : StarkSigner {
```

Comment:
Can we add in the kdocs the definition for the `privateKey` parameter? Mainly to call out that it needs to be in hex format.